### ISSUE TITLE
Write documentation

### ISSUE THREAD

**Issue by scarlehoff** [scarlehoff/pyHepGrid#73]
The readme is more or less complete. It would be nice to reformat it a bit so it can go up to github pages or readthedocs.

**Comment by DWalker487** [comment]
Once this is done (from the looks it's pretty much there anyway), would a minor version release be appropriate, such as v0.9.1 or v0.10? From the looks of it quite a lot of rewriting/refactoring has been done and it might be nice to capture that.

**Comment by scarlehoff** [comment]
Yes, I actually changed the version number to 1.0 in the PR and everything.

**Comment by DWalker487** [comment]
Already on top of it I see :P

**Comment by scarlehoff** [comment]
#76
